main.go

```diff
@@ -342,3 +347,24 @@ func getUserClaims(i kratos_client.Identity, cr hydra_client.OAuth2ConsentReques
 
 	return ret
 }
+
+func setUpPrometheus() *prometheus.MetricsManager {
+	mm := prometheus.NewMetricsManagerWithPrefix("identity-platform-login-ui-operator", "http", "", "", "")
+	mm.RegisterRoutes(
+		"/api/kratos/self-service/login/browser",
+		"/api/kratos/self-service/login/flows",
+		"/api/kratos/self-service/login",
+		"/api/kratos/self-service/errors",
+		"/api/consent",
+		"/consent.html",
+		"/error.html",
+		"/index.html",
+		"/login.html",
+		"/",
+		"",
```

Comment:
What is the difference between these 2?

Reply:
I thought the endpoint was gonna show up differently, but verified that it shows up as the same